back/src/adapters/primary/routers/inclusionConnect/createInclusionConnectedAllowedRouter.ts

```diff
@@ -37,5 +39,22 @@ export const createInclusionConnectedAllowedRouter = (
       ),
   );
 
+  inclusionConnectedSharedRoutes.markPartnersErroredConventionAsHandled(
+    createInclusionConnectedMiddleware(deps.config.jwtPublicKey),
+    async (req, res) =>
+      sendHttpResponse(req, res, () =>
+        match(req.payloads)
+          .with({ inclusion: P.not(P.nullish) }, ({ inclusion }) =>
+            deps.useCases.markPartnersErroredConventionAsHandled.execute(
+              req.body,
+              inclusion,
+            ),
+          )
+          .otherwise(() => {
+            throw new UnauthorizedError();
+          }),
```

Comment:
on peut faire plus simple :
```suggestion
      sendHttpResponse(req, res, () => {
        const inclusion = req.payloads?.inclusion; 
        if(!inclusion) throw new UnauthorizedError();
        deps.useCases.markPartnersErroredConventionAsHandled.execute(
              req.body,
              inclusion,
            )
      }
```